import-export-cli/integration/testutils/policy_testUtils.go

```diff
@@ -0,0 +1,389 @@
+	importedOutput, err := importThrottlePolicy(t, args)
+	assert.Nil(t, err, "Error while importing the Throttling Policy")
+	assert.Contains(t, importedOutput, "Successfully imported")
+	// Give time for newly imported Throttling Policy to get indexed
+	base.WaitForIndexing()
+
+	changeExportedThrottlePolicyFile(t, args)
+	// Import Throttling Policy to env 2 for update conflict
+	output, err := importThrottlePolicy(t, args)
+	assert.Error(t, err, "Importation conflict expected")
+	assert.Contains(t, output, "409", "Unexpected error code")
+	// Give time for newly imported Throttling Policy to get indexed
+	base.WaitForIndexing()
+
+	// Get Throttle Policy from env 2
+	importedPolicy, _ := getThrottlingPolicyByName(t, args, adminUsername, adminPassword, policyName, policyType)
+	// Validate policies are  equal
+	validatePoliciesEquality(t, true, args, importedPolicy)
+}
+
+func ValidateThrottlePolicyImportFailureWithCorruptedFile(t *testing.T, args *PolicyImportExportTestArgs) {
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.DestAPIM.GetEnvName(), args.DestAPIM.GetApimURL(), args.DestAPIM.GetTokenURL())
+
+	// Export policy from env 1
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+
+	path, _, _ := createExportedThrottlePolicyFile(t, args.DestAPIM, apim.AdvancedThrottlePolicyType, true)
+	args.ImportFilePath = path
+	assert.True(t, base.IsFileAvailable(t, args.ImportFilePath))
+
+	//// Import Throttling Policy to dest
+	output, err := importThrottlePolicy(t, args)
+	assert.Error(t, err, "Importation failure expected")
+	assert.Contains(t, output, "500", "Unexpected error code")
+}
+
+func ValidateThrottlePolicyExportFailure(t *testing.T, args *PolicyImportExportTestArgs) {
+	t.Helper()
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+
+	// Export policy from env 1
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	policyName := base.GenerateRandomString() + "Policy"
+
+	output, _ := exportThrottlePolicy(t, policyName, args)
+	assert.Contains(t, output, "Error exporting Throttling Policies", "Exportation error expected")
+}
+
+func changeExportedThrottlePolicyFile(t *testing.T, args *PolicyImportExportTestArgs) {
+	t.Helper()
+	var exportedPolicy utils.ExportThrottlePolicy
+	exportedFile, err := ioutil.ReadFile(args.ImportFilePath)
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = yaml.Unmarshal(exportedFile, &exportedPolicy)
+	if err != nil {
+		t.Fatal(err)
+	}
+	policyData := exportedPolicy.Data
+	policyData[2].Value = base.GenerateRandomString()
+	policyData[3].Value = base.GenerateRandomString()
+	marshaledData, _ := yaml.Marshal(exportedPolicy)
+
+	err = ioutil.WriteFile(args.ImportFilePath, marshaledData, os.ModePerm)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	t.Cleanup(func() {
+		removeExportedThrottlingPolicyFile(t, args.ImportFilePath)
```

Comment:
Do we need this cleanup here? Because inside exportThrottlePolicy we delete the same file, right?

Reply:
fixed with 53f8cc8609cb4711d71c229466ebbd0f9c8f10e5